Notes.md

```diff
@@ -0,0 +1,5 @@
+1. Setup/Installation is Windows OS Specific. Chances are the deploy environment would be a Linux variant.
```

Comment:
Thanks for the review, the Setup/Installation works on Ubuntu cos that's what I use. Though I guess you mean I should use Docker